### ISSUE TITLE
Leading zeros (again)

### ISSUE THREAD

**Issue by AdvancedImagingUTSW** [TheDeanLab/ASLM#836]
```

(navigate) C:\Users\Dean-Lab>navigate
vvvvvvvvvvvvvvvvvvvvvvvvvvv Child Process Traceback vvvvvvvvvvvvvvvvvvvvvvvvvvv
 Traceback (most recent call last):
  File "C:\Users\Dean-Lab\Desktop\ASLM\src\navigate\model\concurrency\concurrency_tools.py", line 501, in _child_loop
    obj = initializer(*initargs, **initkwargs)
  File "C:\Users\Dean-Lab\Desktop\ASLM\src\navigate\model\model.py", line 116, in __init__
    devices_dict = load_devices(
  File "C:\Users\Dean-Lab\Desktop\ASLM\src\navigate\model\device_startup_functions.py", line 1243, in load_devices
    devices["camera"][build_ref_name("_", device["type"], int(camera_serial_number, 8))] = camera
ValueError: invalid literal for int() with base 8: '003209'
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Child Process Traceback ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 vvvvvvvvvvvvvvvvvvvvvvvvvvv Main Process Traceback vvvvvvvvvvvvvvvvvvvvvvvvvvvv
Traceback (most recent call last):
  File "C:\Users\Dean-Lab\miniconda3\envs\navigate\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Dean-Lab\miniconda3\envs\navigate\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Dean-Lab\miniconda3\envs\navigate\Scripts\navigate.exe\__main__.py", line 7, in <module>
  File "C:\Users\Dean-Lab\Desktop\ASLM\src\navigate\main.py", line 113, in main
    Controller(
  File "C:\Users\Dean-Lab\Desktop\ASLM\src\navigate\controller\controller.py", line 161, in __init__
    self.model = ObjectInSubprocess(
  File "C:\Users\Dean-Lab\Desktop\ASLM\src\navigate\model\concurrency\concurrency_tools.py", line 383, in __init__
    assert _get_response(self) == "Successfully initialized"
  File "C:\Users\Dean-Lab\Desktop\ASLM\src\navigate\model\concurrency\concurrency_tools.py", line 438, in _get_response
    raise resp
ValueError: invalid literal for int() with base 8: '003209'

```

**Comment by AdvancedImagingUTSW** [comment]
Addressed in #808 - Still needs testing.

**Comment by AdvancedImagingUTSW** [comment]
Quickly wanted to test this out on a new branch, #836. 

Did some minor formatting for readability.

```
try:
      oct_num = int(camera_serial_number, 8)
      devices["camera"][build_ref_name("_", device["type"], oct_num)] = camera
      devices["camera"][build_ref_name(
                     "_",
                    device["type"],
                    oct_num
                    )] = camera
except ValueError:
      pass
```

On mesoSPIM, with a camera serial number of 003209, the try statement does throw a ValueError, but the camera loads properly.

On CT-ASLM-V1, with a camera serial number of 000007, the try statement is executed properly, and the camera loads properly.

As such, I think it is functioning properly @annie-xd-wang
